charts/scrutiny/charts/web/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
We should probably remove this functionality completely. 
Scrutiny uses a sqlite database under the hood (in addition to the InfluxDB). The sqlite file is used to store device metadata. Sqlite isn't designed for multiple writers, so scaling the web container horizontally wouldn't be safe.

Reply:
I originally thought that multiple instances of the `web` service were possible (each with its own db), but what you're saying makes a lot of sense. I'll get rid of this.